feature-libs/product-configurator/rulebased/components/overview-form/configurator-overview-form.component.html

```diff
@@ -47,3 +19,59 @@ <h2>{{ 'configurator.overviewForm.noAttributeHeader' | cxTranslate }}</h2>
     <p>{{ 'configurator.overviewForm.noAttributeText' | cxTranslate }}</p>
   </div>
 </ng-template>
+
+<ng-template #groups let-overviewGroups="overviewGroups" let-level="level">
+  <ng-container *ngFor="let group of overviewGroups">
+    <div
+      class="cx-group"
+      [class.topLevel]="level === 1"
+      [class.subgroupTopLevel]="level === 1 && group.subGroups?.length > 0"
+    >
+      <h2>
+        <ng-container *cxFeatureLevel="'!3.3'">
```

Comment:
As we want to release this with 4.0 or later, the cxFeatureLevel need to be removed

Reply:
Done